Jenkinsfile

```diff
@@ -51,12 +52,12 @@ pipeline {
       parallel {
         stage('Jasper AN4 O1') {
           steps {
-            sh 'cd examples/asr && CUDA_VISIBLE_DEVICES=0 python jasper_an4.py --amp_opt_level=O1 --num_epochs=40'
+            sh 'cd examples/asr && CUDA_VISIBLE_DEVICES=0 python jasper_an4.py --amp_opt_level=O1 --num_epochs=35 --test_after_training --work_dir=O1'
           }
         }
         stage('Jasper AN4 O2') {
           steps {
-            sh 'cd examples/asr && CUDA_VISIBLE_DEVICES=1 python jasper_an4.py --amp_opt_level=O2 --num_epochs=40'
+            sh 'cd examples/asr && CUDA_VISIBLE_DEVICES=1 python jasper_an4.py --amp_opt_level=O2 --num_epochs=35 --test_after_training --work_dir=O2'
```

Comment:
shouldn't the test outputs be subdirectories of something like test_outputs/* instead of just '01', '02'?

Reply:
It doesn't matter since Jenkins cleans up these directories afterwards
But I should name them jasper-O* so future tests don't use O1 and O2 as logging directories.